x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-attribute-to-ecs@mappings.yaml

```diff
@@ -0,0 +1,27 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Aliases from OpenTelemetry SemConv attribute fields to ECS (and some non-ECS) fields
+  managed: true
+template:
+  mappings:
+    properties:
+      attributes:
+        type: passthrough
+        dynamic: true
+        priority: 10
+        time_series_dimension: true
+        properties:
+          exception.type:
+            type: keyword
+            ignore_above: 1024
+          exception.message:
+            type: keyword
+            ignore_above: 1024
+      error.exception.type:
+        type: alias
+        path: attributes.exception.type
+      error.exception.message:
+        type: alias
+        path: attributes.exception.message
+
```

Comment:
No alias for `handled`, because we send it as `error.exceptin.handled` (by inverting `exception.handled` and attaching it in the enrichment lib) - and that works via passthrough.

Just raising it here to double-check: that's ok, right?